collection-scripts/gather

```diff
@@ -28,3 +29,31 @@ resources+=(clusterautoscaler machineautoscaler)
 for resource in ${resources[@]}; do
     /usr/bin/openshift-must-gather inspect ${resource}
 done
+
+# Collect System Audit Logs
+AUDIT_LOG_PATH="${BASE_COLLECTION_PATH}/audit_logs/"
+mkdir -p ${AUDIT_LOG_PATH}
+/usr/bin/oc adm node-logs --role=master --path=openshift-apiserver/ > ${AUDIT_LOG_PATH}/openshift-apiserver_audit.log
+/usr/bin/oc adm node-logs --role=master --path=kube-apiserver/ > ${AUDIT_LOG_PATH}/kube-apiserver_audit.log
+
+# Collect System Service Logs
+SERVICE_LOG_PATH="${BASE_COLLECTION_PATH}/host_service_logs/"
+MASTER_SERVICE_LOG_PATH="${SERVICE_LOG_PATH}/masters/"
+NODE_SERVICE_LOG_PATH="${SERVICE_LOG_PATH}/nodes/"
+
+# Service Lists
+GENERAL_SERVICES=(kubelet crio)
+MASTER_SERVICES+=${GENERAL_SERVICES[@]}
+NODE_SERVICES+=${GENERAL_SERVICES[@]}
+
+# Collect Master Service Logs
+mkdir -p ${MASTER_SERVICE_LOG_PATH}
+for service in ${MASTER_SERVICES[@]}; do
+    /usr/bin/oc adm node-logs --role=master -u $service > ${MASTER_SERVICE_LOG_PATH}/${service}_service.log
+done
+
+# Collect Worker Service Logs
```

Comment:
I recommend being careful here.  You sure you want to gather this each time?

Reply:
Do you think you will need this to debug CI issues?

Reply:
OK, I split this out (and scoped this to masters), I was not thinking of the concern of having this collect this on 600+ nodes.